lib/graphql_includable.rb

```diff
@@ -28,66 +29,68 @@ def delegate(*methods, args)
   private
```

Comment:
since the below are all class methods this `private` statement doesn't do anything